hpycc/utils/parsers.py

```diff
@@ -141,10 +140,7 @@ def parse_wuid_from_xml(result):
     return wuid.group(0)
 
 
-Schema = namedtuple("Schema", ["name", "is_set", "type"])
-
-
-def parse_schema_from_xml(xml):
+def parse_schema_from_xml(xml, dtype):
```

Comment:
I don't like taking in dtype, it shouldn't be the role of this function

Reply:
removed the dtype stuff from here and added to a new function which is called later.